examples/vanilla/tooltips.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Media Chrome Tooltips</title>
+    <script type="module" src="../../dist/index.js"></script>
+    <style>
+      body {
+        background: #eee;
+        padding: 50px;
+      }
+      
+      .examples {
+        margin-top: 20px;
+      }
+      
+      media-controller {
+        max-width: 300px;
+      }
+      
+      media-control-bar {
+        margin: 10px;
+      }
+
+      code {
+        color: maroon;
+      }
+    </style>
+  </head>
+  <body>     
+    <main>
+      <h1>Tooltips</h1>
+      
+      <h2>On their own (no arrows when <code>position="none"</code>)</h2>
+
+      <div>
+        <media-tooltip position="none">Settings</media-tooltip>
+        <br><br>
+        <media-tooltip position="none">Play</media-tooltip>
+        <br><br>
+        <media-tooltip position="none">Pause</media-tooltip>
+      </div>
+
+      <h2>As part of button controls</h2>
+
+      <media-controller>
+        <video
+          slot="media"
+          src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/high.mp4"
+          muted
+        ></video>
+        <media-control-bar>
+          <media-play-button></media-play-button>
+        </media-control-bar>
+      </media-controller>
+
+      <h2>Overriding tooltip label <small>(and constraining x position)</small></h2>
+      <media-controller>
+        <video
+          slot="media"
+          src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/high.mp4"
+          muted
+        ></video>
+        <media-control-bar style="display: flexbox; justify-content: space-between;">
+          <media-play-button>
+            <span slot="tooltip-play"><strong>Play</strong> the movie! <img src="/examples/vanilla/images/spacebar.svg" width="12"></span>
+            <span slot="tooltip-pause">Please <i>stop</i>... <img src="/examples/vanilla/images/spacebar.svg" width="12"></strong></span>
+          </media-play-button>
+          <media-play-button>
+            <span slot="tooltip-play">Play......</span>
+            <span slot="tooltip-pause">Pause........</span>
+          </media-play-button>
+        </media-control-bar>
+      </media-controller>
+
+      <h2>Hiding with CSS</h2>
+
+      <media-play-button style="--media-tooltip-display: none;"></media-play-button>
```

Comment:
This example completely hides the tooltip where as `tooltipposition="none"` leaves the tooltip visible but doesn't position it in any way (position `relative` essentially). Where you imagining that ``tooltipposition="none"` would also `display: none` the tooltip? something I hadn't considered!

Reply:
Yeah, that's what I meant by the "none" option. The css var works but it's a tad bit advanced for how important that feature is, and the attribute option will surface the feature in docs better. Otherwise I can't see much value in just an unpositioned tooltip. 

Open to other naming/setting directions for that, but it felt nice to not introduce another attribute.

Reply:
I think the value in an un-positioned tooltip is if you want to be in control of positioning yourself. If a 'none' position hides the tooltip, there's no way to opt out of the positioning that is baked in (position absolute, transforms, etc) unless we add a lot more css vars. There's also something slightly odd to me about a `position` property controlling visibility, but I don't feel particularly strongly about any of it so happy to go in either direction. Maybe @luwes or @cjpillsbury can offer a tie breaker? 🙂

Reply:
I would also not overload the `tooltipposition` attr with a hiding functionality.

Weren't we going to add an easy attribute to change the text of the tooltips within the button. 
Could it be an empty `tooltip=""` attribute?

Reply:
I've pushed up a tiny CSS change that would allow people to just use the standard global `hidden` attribute e.g. `<media-tooltip hidden>`, is there a reason not to use that? admitted I haven't used it myself in a long time, but it seems to be using it as intended.

Reply:
`hidden` works for a stand-alone tooltip 👍 We also need an attribute for the media-play-button etc.

To Wes's point, I do think we need a `tooltip=""` attribute for these buttons as an easy way to set that. I know the pushback has been "two ways of doing the same thing", but @mmcc commented that changing the content and swapping an element are really two different use cases, and I agree with that.

From there, I don't have a strong opinion if it's `tooltip=""` or `notooltip`. Either one. `notooltip` feels a little cleaner since we're going to default to them being on, but `tooltip=""` means one less attr.